ddlite.py

```diff
@@ -1157,14 +1158,28 @@ def get_predicted_probability(self, subset=None):
     Get array of predicted probabilities (continuous) given weights
     Return either all candidates, a specified subset, or only validation/test set
     """
-    return odds_to_prob(self.get_log_odds(subset))
+    return self._prob_sub(self.probabilities, subset)
+ 
+  def get_lf_predicted_probability(self, subset=None):
+    """
+    Get array of predicted probabilities (continuous) given LF weights
+    Return either all candidates, a specified subset, or only validation/test set
+    """
+    return self._prob_sub(self.lf_marginals, subset)
  
   def get_predicted(self, subset=None):
     """
     Get the array of predicted (boolean) variables given weights
     Return either all variables, a specified subset, or only validation/test set
     """
-    return np.sign(self.get_log_odds(subset))
+    return np.sign(self.get_predicted_probability(subset) - 0.5)
+    
+  def get_lf_predicted(self, subset=None):
+    """
+    Get the array of predicted (boolean) variables given LF weights
+    Return either all variables, a specified subset, or only validation/test set
+    """
+    return np.sign(self.get_lf_predicted_probability(subset) - 0.5)
```

Comment:
Still think these fn names are misleading, again maybe something like `get_training_marginals` and `get_training_predicted`?